course-offline/assets/course-offline.ts

```diff
@@ -27,6 +27,8 @@ export interface OCWWindow extends Window {
 
 declare let window: OCWWindow
 
+const initVideoJS = _.once(() => (window.videojs = videojs))
+
```

Comment:
Is there a reason to make this change? I don't think the `window.videojs = videojs` assignment was problematic.